install-dev/upgrade/sql/1.7.6.2.sql

```diff
@@ -0,0 +1,50 @@
+SET SESSION sql_mode = '';
+SET NAMES 'utf8';
+
+INSERT INTO `PREFIX_category_lang` SELECT
+`entity`.`id_category`, 1, `entity`.`id_lang`, `entity`.`name`, `entity`.`description`, `entity`.`link_rewrite`, `entity`.`meta_title`, `entity`.`meta_keywords`, `entity`.`meta_description`
+FROM `PREFIX_category_lang` entity
+LEFT JOIN `PREFIX_category_lang` entity2 ON `entity2`.`id_shop` = 1 AND `entity`.`id_category` = `entity2`.`id_category`
```

Comment:
I don't get it, you force the `id_shop` to `1`, but select values from `entity2.id_shop = 2 AND null`
So no rows is inserted with id_shop values different from 1, and even in the select how can id_shop be both equal to 1 and null

Sure, I'm not a champion in SQL ^^ But I'm not the only one so maybe you could add a comment to explain what the query does?

Reply:
I'm looking for rows that haven't data with id_shop = 1.
The `LEFT JOIN` combined with a `WHERE NULL` check for nonexistent rows.
This means `Give me id_category that have id_shop != 1 and don't have a row with the same id_category and id_shop 1`.

This kind of SQL request are often used for finding duplicates.

Reply:
Alright I finally understood it ^^ Thanks for learning a new SQL technique